src/snappy/snappymodule.cc

```diff
@@ -33,7 +33,6 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <snappy-c.h>
 #include "crc32c.h"
 
-#define MODULE_VERSION "0.4.1"
```

Comment:
Note: this incorrect string is currently available as `snappy._snappy.__version__`